packages/grafana-data/src/dataframe/StreamingDataFrame.ts

```diff
@@ -1,21 +1,38 @@
+import { AlignedData } from 'uplot';
+
+import { DataFrame, Field, FieldDTO, FieldType, Labels, parseLabels, QueryResultMeta, renderLegendFormat } from '..';
+import { join } from '../transformations/transformers/joinDataFrames';
+
 import {
-  DataFrame,
   DataFrameJSON,
   decodeFieldValueEntities,
-  Field,
-  FieldDTO,
   FieldSchema,
-  FieldType,
   guessFieldTypeFromValue,
-  Labels,
-  parseLabels,
-  QueryResultMeta,
   toFilteredDataFrameDTO,
-} from '@grafana/data';
-import { join } from '@grafana/data/src/transformations/transformers/joinDataFrames';
-import { StreamingFrameAction, StreamingFrameOptions } from '@grafana/runtime/src/services/live';
-import { renderLegendFormat } from 'app/plugins/datasource/prometheus/legend';
-import { AlignedData } from 'uplot';
+} from '.';
+
+/**
+ * Indicate if the frame is appened or replace
+ * From @grafana/runtime/src/services/live
```

Comment:
I am copying these (simple) types from @grafana/runtime. I found no better option but maybe I am missing something?

Reply:
rather than copy -- can we "move" them to data.  essentially export here, then re-export in grafana data